winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Columns in WinUI Kanban control | Syncfusion
+description: Learn here all about configuring and customizing columns in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Column in WinUI Kanban Control
+
+The WinUI Kanban control organizes its layout into columns, representing distinct stages of a workflow or process. Common stages include To-Do, In-Progress, Testing, Validation, and Completed.
+
+## Customizing column width
+
+Columns are smartly sized by default to ensure optimal arrangement and readability of card elements. However, you can customize column widths to suit your specific requirements by using `MinimumColumnWidth` and `MinimumColumnWidth` properties of `SfKanban`.
+
+{% tabs %}
+{% highlight XAML hl_lines="3 4" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 MinimumColumnWidth="300"
+                 MinimumColumnWidth="350">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.MinimumColumnWidth = 300;
+this.kanban.MaximumColumnWidth = 350;
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also define the exact width for the columns using `ColumnWidth` property of `SfKanban`.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 ColumnWidth="400">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.ColumnWidth = 400;
```

Comment:
Add ItemsSource code also in all places.

If we given 400, Is UI looks good? because Image should be based on code snippet added.

Reply:
Added ItemsSource and changed columnwidth to 320